rasa/nlu/extractors/entity_synonyms.py

```diff
@@ -13,16 +12,17 @@
 from rasa.nlu.extractors.extractor import EntityExtractorMixin
 import rasa.utils.io
 import rasa.shared.utils.io
+from rasa.nlu.extractors._entity_synonyms import EntitySynonymMapper
 from rasa.engine.storage.resource import Resource
 from rasa.engine.storage.storage import ModelStorage
 
+# This is a workaround around until we have all components migrated to `GraphComponent`.
+EntitySynonymMapper = EntitySynonymMapper
```

Comment:
It looks like this is not up to date with main. Once you update, this should not touch any Research-owned code and you don't need a review from Research (unless it still does).